snapcraft/services/__init__.py

```diff
@@ -16,25 +16,27 @@
 
 """Snapcraft services."""
 
-from snapcraft.services.assertions import Assertion
-from snapcraft.services.init import Init
-from snapcraft.services.lifecycle import Lifecycle
-from snapcraft.services.package import Package
-from snapcraft.services.provider import Provider
-from snapcraft.services.confdbs import Confdbs
-from snapcraft.services.remotebuild import RemoteBuild
-from snapcraft.services.service_factory import (
+from .assertions import Assertion
```

Comment:
this is a curious change, what prompted it?

Reply:
First I went to add the Project service and got annoyed it wasn't alphabetized.  Then I went whole hog and did the relative imports for readability.  

None of that is strictly necessary. I wouldn't have added it if I knew how massive this PR was going to get.